cloud_functions/strava_auth_initiate/main.py

```
@@ -1,167 +1,165 @@
+        error=str(e),
+        level="ERROR",
+        traceback=traceback.format_exc(),
+    )
+    raise
+
+
+@https_fn.on_request(
+    cors=options.CorsOptions(cors_origins=["*"], cors_methods=["POST", "OPTIONS"])
+)
+@functions_framework.http
+def strava_auth_initiate(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `strava_auth_initiate`

(ANN201)